### DISCUSSION TITLE
Display vector graphic from ide

### DISCUSSION THREAD

**Discussion by SevenChords** [cduck/drawSvg#113] [Q&A] [answered]
is there a way to display the generated svg drawings from an ide instead of a jupyter notebook?

**Comment by cduck** [comment, marked as answer]
Many IDEs support Jupyter's inline display functionality.  Look for a tutorial that explains how to enable Jupyter widgets (or matplotlib plots) to be displayed inline for your specific IDE.  If other Jupyter widgets display in your IDE but drawsvg drawings still do not, please [open an issue](https://github.com/cduck/drawsvg/issues/new/choose).
